.circleci/config.base.yml

```diff
@@ -305,6 +305,32 @@ jobs:
           path: ~/repo/packages/amplify-migration-tests/amplify-migration-reports
     working_directory: ~/repo
 
+  amplify_migration_tests_v10:
+    <<: *defaults
+    environment:
+      AMPLIFY_PATH: /home/circleci/.amplify/bin/amplify
```

Comment:
for v10 migration tests, and any v10+; the amplify -g installation is at the .amplify/bin/amplify path